wgsl/index.bs

```diff
@@ -22,9 +22,11 @@ Ignored Vars: i, c0, e, e1, e2, e3, edge, eN, p, s1, s2, sn, AS, AM, N, newbits,
 !Participate: <a href="https://github.com/gpuweb/gpuweb/issues/new?labels=wgsl">File an issue</a> (<a href="https://github.com/gpuweb/gpuweb/issues?q=is%3Aissue+is%3Aopen+label%3Awgsl">open issues</a>)
 !Tests: <a href=https://github.com/gpuweb/cts/tree/main/src/webgpu/shader/>WebGPU CTS shader/</a>
 
+Editor: Alan Baker, Google https://www.google.com, alanbaker@google.com
+Editor: Mehmet Oguz Derin, mehmetoguzderin@mehmetoguzderin.com
```

Comment:
@mehmetoguzderin I don't know if you want an organization or if you have a W3C ID.

Reply:
@dneto0 My login is MehmetOguzDerin which resolves to this URL when signed in: https://www.w3.org/users/101130

If no affiliation is possible, that would be alright with me.